### PR TITLE
CI: disable ARM build

### PR DESCRIPTION
**Pull request by jbrockmendel**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

xref #36719

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -46,9 +46,10 @@ matrix:
     - env:
         - JOB="3.7" ENV_FILE="ci/deps/travis-37.yaml" PATTERN="(not slow and not network and not clipboard)"
 
-    - arch: arm64
-      env:
-        - JOB="3.7, arm64" PYTEST_WORKERS=1 ENV_FILE="ci/deps/travis-37-arm64.yaml" PATTERN="(not slow and not network and not clipboard and not arm_slow)"
+    # Disabled 2020-09-29 due to timeoutshttps://github.com/pandas-dev/pandas/issues/36719
```
Comment (jreback) [inline .travis.yml:49]: instead can we move it to allowed failures again

Comment (jbrockmendel) [reply .travis.yml:49]: wont that still run and cause the Travis turnaround to be super-slow?

Comment (jreback) [reply .travis.yml:49]: it will turn the build green as soon as the non allow failure builds succeed

**Comment by jbrockmendel** [comment]
could use some help with the config.  tried to match the older version, but looks like it isnt running the allow-fail build

**Comment by jreback** [comment]
> could use some help with the config. tried to match the older version, but looks like it isnt running the allow-fail build

just leave the original AND the new and it will work; e.g. it is a copy that's *also* in allow_failures.

**Pull request merged**